### ISSUE TITLE
How to install on Kibana 7.10.2

### ISSUE THREAD

**Issue by reski-rukmantiyo** [beshu-tech/readonlyrest-docs#83]
Hi Team,

I'm using ELK with version 7.10.2 and after I execute kibana-plugin installation, i got this error

> bash-4.4$ bin/kibana-plugin install "https://api.beshu.tech/download/trial?edition=kbn_enterprise&email=xxxxxx"
Attempting to transfer from https://api.beshu.tech/download/trial?edition=kbn_enterprise&email=xxxxxxx
Transferring 19357504 bytes....................
Transfer complete
Retrieving metadata from plugin archive
Plugin installation was unsuccessful due to error "No kibana plugins found in archive"
bash-4.4$ bin/kibana-plugin list

How to use this plugins?

**Comment by sscarduzio** [comment]
At the moment we don't have a Kibana plugin for 7.10.x. We have stable 7.8.1 and a beta for 7.9.3

**Comment by reski-rukmantiyo** [comment]
Thanks for your clarification. So it means I can use it as long as i’m using 7.8.1 or 7.93 ?
Thanks

Get Outlook<https://aka.ms/sdimjr> for iPad OS
________________________________

**Comment by sscarduzio** [comment]
Yes of course, have a look at this single command demo (using Docker)

https://github.com/sscarduzio/ror-docker-demo

Let me know if you need help
